docs/tornjak-helm-chart.md

```diff
@@ -0,0 +1,139 @@
+# Deploying Tornjak via SPIFFE Helm Charts Hardened
+
+## Overview
+
+You can deploy **Tornjak** using the [SPIFFE Helm Charts Hardened](https://github.com/spiffe/helm-charts-hardened)
+repository. 
+
+This guide walks you through deploying both the frontend and backend of Tornjak with Direct Access, 
+using Helm charts in a local Kubernetes environment via Minikube. By the end, you’ll have a working instance 
+of SPIRE integrated with Tornjak for easier visibility and management of your SPIFFE identities. 
+
+## Prerequisites
+
+Make sure you have the following installed on your system:
+
+- [Minikube](https://minikube.sigs.k8s.io/docs/start/?arch=%2Fmacos%2Fx86-64%2Fstable%2Fbinary+download)
+- [Helm](https://helm.sh/docs/intro/install/)
+
+---
+
+## Step-by-Step Deployment
+
+### 1. Start Minikube
+
+```bash
+minikube start
+``` 
+
+Once minikube is up you can continue to the next step.
+
+---
+
+### 2. Clone the Helm Charts Repository 
+
+Clone the `helm-charts-hardened` repository if you haven't already, then navigate 
+into it and create a new branch. 
+
+```bash
+git clone https://github.com/spiffe/helm-charts-hardened.git
+cd helm-charts-hardened 
+git checkout -b your-branch
+```
+
+You can name the branch whatever you like.
```

Comment:
This part will be unnecessary if you follow the below comment with putting this tutorial doc in a separate folder. The helm install commands below I believe do not require the repo to be cloned